lib/pages/chat/reply_display.dart

```diff
@@ -65,16 +65,22 @@ class _EditContent extends StatelessWidget {
           color: Theme.of(context).colorScheme.primary,
         ),
         Container(width: 15.0),
-        Text(
-          event.calcLocalizedBodyFallback(
-            MatrixLocals(L10n.of(context)!),
-            withSenderNamePrefix: false,
-            hideReply: true,
-          ),
-          overflow: TextOverflow.ellipsis,
-          maxLines: 1,
-          style: TextStyle(
-            color: Theme.of(context).textTheme.bodyMedium!.color,
+
+        // #Pangea
```

Comment:
Why does this need to be added (was there an overflow issue)?

Reply:
Yeah, long replies could overflow the display. I figured it was an easy enough fix.